doc/01-About.md

```diff
@@ -0,0 +1,27 @@
+# Icinga Notifications Web
+
+!!! warning
+This is an early beta version for you to try, but do not use this in production. There may still be severe bugs.
+At the moment, we don't provide any support for this module.
+
+Icinga Notifications is a set of components that processes received events from miscellaneous sources, manages
+incidents and forwards notifications to predefined contacts, consisting of:
+
+* An [Icinga Notifications Daemon](https://github.com/Icinga/icinga-notifications), which receives events and sends
+  notifications.
+* The Icinga Notifications Web module (this repository), which provides graphical configuration and further
+  processing of the data collected by the daemon.
```

Comment:
To avoid mentioning *daemon* again, drop this. Web doesn't *process* as in the same meaning in which the daemon does it.